.travis.yml

```diff
@@ -22,6 +22,9 @@ matrix:
 
 before_install:
   - docker run -d -p 5984:5984 --net=host --name couch apache/couchdb:2.3
+  - curl -sL https://deb.nodesource.com/setup_8.x | sudo bash -
+  - sudo apt install nodejs
```

Comment:
Can we use `nvm` and `node 8.12.0` as https://docs.travis-ci.com/user/reference/xenial/#javascript-and-nodejs-support says that they are installed by default?